controllers/webhooksController.js

```diff
@@ -15,7 +15,10 @@ module.exports = (() => {
       const endpointSecret = process.env.END_POINT_STRIPE;
       const sig = req.headers['stripe-signature'];
       let event;
-      let session = '';
+      const checkoutCompleted = 'checkout.session.completed';
```

Comment:
I like the idea of making costs, but maybe we should move it to a JSON full of all the webhooks from Stripe that we want make some cases that we hold in a config folder